test/plugin/scenarios/rocketmq-5-grpc-scenario/src/main/java/test/apache/skywalking/apm/testcase/rocketmq/client/java/controller/CaseController.java

```diff
@@ -0,0 +1,151 @@
+                    .enableSsl(false)
+                    .build();
+            // start producer
+            Producer producer = provider.newProducerBuilder()
+                    .setClientConfiguration(clientConfiguration)
+                    .build();
+            System.out.printf("Provider Started.%n");
+
+            // send msg
+            Message message = provider.newMessageBuilder()
+                    // Set topic for the current message.
+                    .setTopic(TOPIC)
+                    // Message secondary classifier of message besides topic.
+                    .setTag(TAG)
+                    // Key(s) of the message, another way to mark message besides message id.
+                    .setKeys("KeyA")
+                    .setBody("This is a normal message for Apache RocketMQ".getBytes(StandardCharsets.UTF_8))
+                    .build();
+            SendReceipt sendReceipt = producer.send(message);
+            System.out.printf("%s send msg successfully, message: %s%n", new Date(), sendReceipt);
+
+            // start consumer
+            Thread thread = new Thread(new Runnable() {
+                @Override
+                public void run() {
+                    try {
+                        FilterExpression filterExpression = new FilterExpression(TAG, FilterExpressionType.TAG);
+                        PushConsumer consumer = provider.newPushConsumerBuilder()
+                                .setClientConfiguration(clientConfiguration)
+                                .setConsumerGroup(GROUP)
+                                .setSubscriptionExpressions(Collections.singletonMap(TOPIC, filterExpression))
+                                .setMessageListener(new MyConsumer())
+                                .build();
+                        System.out.printf("Consumer Started.%n");
+                    } catch (Exception e) {
+                        log.error("consumer start error", e);
+                    }
+                }
+            });
+            thread.start();
+        } catch (Exception e) {
+            log.error("testcase error", e);
+        }
+        return SUCCESS;
+    }
+
+    @RequestMapping("/healthCheck")
+    @ResponseBody
+    public String healthCheck() throws Exception {
+        System.setProperty(MixAll.ROCKETMQ_HOME_ENV, this.getClass().getResource("/").getPath());
+        String[] subArgs = new String[]{
+                "updateTopic",
+                "-n",
+                nameServer,
+                "-c",
+                "DefaultCluster",
+                "-t",
+                "TopicTest"};
+        MQAdminStartup.main(subArgs);
+
+        subArgs = new String[]{
+                "updateSubGroup",
+                "-n",
+                nameServer,
+                "-c",
+                "DefaultCluster",
+                "-g",
+                "group1"};
+        MQAdminStartup.main(subArgs);
+        System.out.printf("HealthCheck Provider Started.%n");
```

Comment:
No system out in codes is allowed.
Health check should include provider and consumer creation, right?
And the topic, provider and consumer should be created once.

Reply:
i find system out in many code. If the rule is like this, I will remove it

i will add create provider  in Health check